docs/ops/activation/Exp_1.md

```diff
@@ -6,12 +6,43 @@
 
 **Short description**: Exponential element-wise activation function.
 
-**Attributes**: has no attributes
+**Detailed description**
 
-**Inputs**:
+*Exp* performs element-wise exponential activation function on a given input tensor. The mathematical formula is as follows:
 
-*   **1**: Input tensor x of any floating point type. Required.
+\f[
+exp(x) = e^{x}
+\f]
 
-**Outputs**:
+**Attributes**: *Exp* operation has no attributes.
 
-*   **1**: Result of Exp function applied to the input tensor *x*. Floating point tensor with shape and type matching the input tensor.
+**Inputs**
+
+*   **1**: A tensor of type `T` and arbitrary shape. **Required**.
+
+**Outputs**
+
+*   **1**: The result of element-wise *Exp* function applied to the input tensor. A tensor of type `T` and same shape as input tensor.
```

Comment:
of type `T` and same shape ---> of type `T` and the same shape

Reply:
Done